templates/octaviaapi/config/octavia.conf

```diff
@@ -74,37 +75,44 @@ forward_all_logs=True
 disable_local_log_storage=False
 [keepalived_vrrp]
 [service_auth]
-project_domain_name=Default
-project_name=service
-user_domain_name=Default
-password=FIXMEpw3
-username=octavia
-auth_type=password
-auth_url={{ .KeystonePublicURL }}/v3
-region_name=regionOne
+# project_domain_name=Default
+# project_name=service
+# user_domain_name=Default
+# password=FIXMEpw3
+# username=octavia
+# auth_type=password
+# auth_url={{ .KeystonePublicURL }}/v3
+# region_name=regionOne
 [nova]
-region_name=regionOne
-endpoint_type=internalURL
-enable_anti_affinity=True
+auth_url = {{ .KeystonePublicURL }}
+auth_type = password
+project_domain_name = Default
+user_domain_name = default
+region_name = regionOne
+username = nova
 [cinder]
-region_name=regionOne
-endpoint_type=internalURL
+# region_name=regionOne
+# endpoint_type=internalURL
 [glance]
-region_name=regionOne
-endpoint_type=internalURL
+# region_name=regionOne
+# endpoint_type=internalURL
 [neutron]
-region_name=regionOne
-endpoint_type=internalURL
+auth_url = {{ .KeystonePublicURL }}
+auth_type = password
+project_domain_name = Default
+user_domain_name = default
+region_name = regionOne
+username = nova
```

Comment:
Should the `username` be `neutron` here?

Reply:
Thanks for catching that. It should probably be neutron.